tests/platform_tests/api/test_chassis_fans.py

```diff
@@ -190,13 +220,26 @@ def test_get_fans_speed_tolerance(self, duthosts, enum_rand_one_per_hwsku_hostna
         self.assert_expectations()
 
     def test_set_fans_speed(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, platform_api_conn):
+
+        fans_skipped = 0
         duthost = duthosts[enum_rand_one_per_hwsku_hostname]
         if duthost.facts["asic_type"] in ["cisco-8000"]:
             target_speed = random.randint(40, 60)
         else:
             target_speed = random.randint(1, 100)
 
         for i in range(self.num_fans):
+            speed_controllable = self.get_fan_facts(duthost, i, True, "speed", "controllable")
+            if not speed_controllable:
```

Comment:
Same comment as my previous comment for platforms that does not support capability what happens? it will also be skipped? then that would be wrong right?  it should check if platform does not support capability query yet should not be skipped??

Reply:
never mind my comments... I see there is a default value involved with the call so it should be fine...